src/test/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationImplDcTest.java

```diff
@@ -108,7 +108,7 @@ public void testBalancesAdjustmentWithDifferentStateId() {
         BalanceComputationResult result = balanceComputation.run(testNetwork1, newStateId, parameters).join();
 
         assertEquals(BalanceComputationResult.Status.SUCCESS, result.getStatus());
-        assertEquals(2, result.getIterationCount());
+        assertEquals(5, result.getIterationCount());
```

Comment:
Is it normal that so many iterations are needed in DC now ?

Reply:
@annetill any idea what could have changed in DC calculation? Distributed slack has been added between 0.8 and 0.9?

Reply:
Yes, I have implemented the DC slack distribution, see the release notes https://github.com/powsybl/powsybl-open-loadflow/releases. You want me to deep into this through the logs?

Reply:
Seb, could you try to put off the distributed slack?